sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -584,7 +584,7 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
             podSpec.getSeccompUnconfined(),
             podSpec.getSeccompProfileName());
 
-      if (podSpec.getSeccompUnconfined()) {
+      if (podSpec.getSeccompUnconfined() != null && podSpec.getSeccompUnconfined()) {
```

Comment:
Lets remove this change from this PR, we'll issue a seperate PR to track breakages due to upgrades relating to Seccomp. (https://github.com/mesosphere/dcos-commons/pull/3070)